internal/worker/controlleragentconfig/worker.go

```diff
@@ -107,6 +133,24 @@ func newWorker(cfg WorkerConfig, internalStates chan string) (*configWorker, err
 	return w, nil
 }
 
+func (w *configWorker) registerHandlers(r *mux.Router) {
+	r.HandleFunc("/reload", w.reloadHandler).
+		Methods(http.MethodGet)
```

Comment:
Was the choice of GET discussed? I think POST is more intuitive, and might be extended in future such that the body is pertinent...

Reply:
I don't believe me or Simon had a strong reason to use GET here, I'll change it to POST.

I'll also change it in the PR on the controller charm.

Reply:
Is it worth checking that the body is empty, and throwing an error is it is not?

Reply:
Not at this point.